lib/config/fileconf_test.go

```diff
@@ -137,6 +139,19 @@ func TestAuthSection(t *testing.T) {
 				cfg["auth_service"].(cfgMap)["web_idle_timeout"] = "potato"
 			},
 			expectError: require.Error,
+		}, {
+			desc: "Proxy ping interval",
+			mutate: func(cfg cfgMap) {
+				cfg["auth_service"].(cfgMap)["proxy_ping_interval"] = "10s"
+			},
+			expectError:             require.NoError,
+			expectProxyPingInterval: requireEqual(types.Duration(10 * time.Minute)),
```

Comment:
Maybe I'm missing something, but `proxy_ping_interval` is set to 10 SECONDS, and here we expect 10 MINUTES?

Reply:
You're right. The `expect` part was missing. Thanks!